app/src/main/java/ch/epfl/favo/user/UserUtil.java

```diff
@@ -114,4 +115,7 @@ public void retrieveUserRegistrationToken(User user) {
               collection.updateDocument(user.getId(), notifMap);
             });
   }
+  public void setCollectionWrapper(CollectionWrapper collectionWrapper){
```

Comment:
is this for testing?

Reply:
Yes. The issue is that the collection wrapper was only declared when instantiating the single instance. So if we want to change the behaviour of the collection wrapper later on for testing we can't